.github/wiki/Fastlane.md

```diff
@@ -1,150 +1,146 @@
 
 Example:
 
 ```
-$ bundle exec fastlane register_new_device
+$ bundle exec fastlane registerNewDevice
 ```
 
 ### Build and upload the application
 
-So as to build and upload the application to distribution tools, like Firebase or App Store Connect, please use these lanes:
+To build and upload the application to distribution tools, like Firebase or App Store Connect, please use these lanes:
 
-| `build_and_upload_staging_app` | `build_and_upload_production_app` | `build_and_upload_appstore_app` |
-|---|---|---|
-| To upload the Staging build and Staging dSYM file to Firebase. | To upload the Production build and Production dSYM file to Firebase. | To upload the Production build to App Store and Testflight. |
+| `buildStagingAndUploadToFirebase` | `buildProductionAndUploadToFirebase` | `buildAndUploadToAppStore` | `buildAndUploadToTestFlight` |
+|---|---|---|---|
+| To upload the Staging build and Staging dSYM file to Firebase. | To upload the Production build and Production dSYM file to Firebase. | To upload the Production build to App Store. | To upload the Production build to TestFlight. |
 
 Example: 
 
 ```
-$ bundle exec fastlane build_and_upload_staging_app
-$ bundle exec fastlane build_and_upload_production_app
-$ bundle exec fastlane build_and_upload_appstore_app
+$ bundle exec fastlane buildStagingAndUploadToFirebase
+$ bundle exec fastlane buildProductionAndUploadToFirebase
+$ bundle exec fastlane buildAndUploadToAppStore
+$ bundle exec fastlane buildAndUploadToTestFlight
 ```
 
 > See more:
 > 
-> - [Gymfile](#gymfile)
-> - [BuildManager.rb](#buildmanagerrb)
-> - [DistributionManager.rb](#distributionmanagerrb)
+> - [Build.swift](#buildswift)
+> - [Distribution.swift](#distributionswift)
 
 
 ## Matchfile
 
-Define the basic information to synchronize the certificates and profiles across the development team.
+Define the essential information to synchronize the certificates and profiles across the development team.
 
 > See more:
 > 
-> - [MatchManager.rb](#matchmanagerrb)
+> - [Match.swift](#matchswift)
 > - [Synchronize the provisioning profile](#synchronize-the-provisioning-profile)
 
-## Gymfile
-
-Store defaults parameters when triggering a new build.
-
 ## Constants folder
 
-### Constants.rb
+### Constant.swift
 
-Contains the key/value pairs of constants that will be used during the development and release process.
+It contains the key/value pairs of constants used during the development and release process.
 
-### Environments.rb
+### Secret.swift
 
-Contains the key/value pairs of environment variables that will be used during the development and release process.
+It contains the key/value pairs of environment variables used during the development and release process.
 
 ## Managers folder
 
-### BuildManager.rb
+### Build.swift
 
-The `BuildManager` helps build and sign the application. There are two main functions:
+The `Build` helps build and sign the application. There are two main functions:
```

Comment:
```suggestion
The `Build` helps building and signing the application. There are two main functions:
```

Reply:
I will keep the pure verb. Ref: https://www.oxfordlearnersdictionaries.com/definition/american_english/help_1

![Screenshot 2023-01-06 at 13 36 35](https://user-images.githubusercontent.com/24598204/210944516-a043c077-684f-4804-91f4-65f8f8819051.png)

![Screenshot 2023-01-06 at 13 37 09](https://user-images.githubusercontent.com/24598204/210944569-84a2f0cf-14a6-4c2c-b3bd-42b46158c0b2.png)

Reply:
I think you can start this sentence with no `The`.
```suggestion
`Build.swift` helps build and sign the application. There are two main functions:
```